client/main.lua

```diff
@@ -66,8 +66,13 @@ AddTextEntry('FE_THDR_GTAO', mapText)
 
 CreateThread(function()
     for _, v in pairs(GetVehiclesByName()) do
-        if v.hash and v.name then
-			AddTextEntryByHash(v.hash, v.name)
-		end
+        if v.model and v.name then
+            local gameName = GetDisplayNameFromVehicleModel(v.model)
+            if gameName and gameName ~= "CARNOTFOUND" then
+                AddTextEntryByHash(joaat(gameName), v.name)
+            else
+                lib.print.warn('Could not find gameName value in vehicles.meta for vehicle model %s', v.model)
```

Comment:
Maybe instead, you should collect the models into a table and do one print?

Reply:
looks way better this way imo 🤷🏻